src/pages/RequestInvite/index.tsx

```diff
@@ -127,15 +140,16 @@ export class RequestInvite extends React.Component<Props, State> {
       return;
     }
 
-    const requestOp = getRequestInviteOperation(
-      creatorMid,
-      this.props.loggedInFirebaseUser.uid,
-      requestingFromMember.mid,
-      requestingFromMember.uid,
-      fullName
-    );
     try {
-      await this.props.postOperation(requestOp);
+      if (this.props.requestInvite) {
```

Comment:
In the future, we should probably disable the button if it isn't present, which will be easy once this component uses the standardized `Button` component—but that can be a different PR.